app/(user)/profile/components/CandidatesSection.js

```diff
@@ -0,0 +1,53 @@
+/**
+ *
+ * CandidatesSection
+ *
+ */
+
+import React, { useContext } from 'react';
+import gpFetch from 'gpApi/gpFetch';
+import gpApi from 'gpApi';
+import { cookies } from 'next/headers';
+import Link from 'next/link';
+import CandidateCard from '@shared/candidates/CandidateCard'
+
+async function fetchCandidates(params) {
+    const nextCookies = cookies();
+    const api = { ...gpApi.follow.list, authToken: nextCookies.get('token').value };
```

Comment:
let's crate a function called "GetServerToken" on /helpers/userHelper.js that reads the cookies from and returns 
```
nextCookies.get('token').value
```